### PR TITLE
Added documentation to imdb.py

### LINKED ISSUE #10: Adding documentation to imdb.py
**Actual Behaviour**

The code seems messy for a newbie to understand , it requires documentation and proper structure so that the code looks clean and can be understood easily.

**Expected Behaviour**

The feature will make the code self explanatory and allow users/contributors to understand quickly. 

**Steps to reproduce it**

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by AJ-54**
Fixes #10 
The documentation now makes code look more clean and easily understandable.

### CODE REVIEW

File: imdb.py [hunk 1]
```diff
@@ -5,24 +5,43 @@
 import bs4
```
Comment (akshatnitd) [inline imdb.py]: Write the inline documentation a line before the actual command, instead of after the command.

Comment (AJ-54) [reply imdb.py]: like for example , the line 8 documentation should be made above  the current line 8 command??

Comment (AJ-54) [reply imdb.py]: @akshatnitd I have done the proper documentation. 
Kindly look and suggest changes if any.
Thanks

File: imdb.py [hunk 2]
```diff
@@ -4,56 +4,80 @@
...
 
 
 
 def info_movie():
-    name = raw_input('\nEnter the title of the movie: ')
-    t = name.replace(' ','%20')
+	
+    name = raw_input("\n Enter the title of the movie: ")
+	
+	#replaces all whitespaces with string '%20'
```
Comment (akshatnitd) [inline imdb.py]: The indentation of documentation and commands a-is not same. Also, leave a space after `#` and write the comments in Sentence Case.

Comment (AJ-54) [reply imdb.py]: but in my local file , the indentation is perfectly fine ( i checked on vscode and notepad++). How does it changes here. And likewise is there any biasness of one editors's indentation over other?

Comment (AJ-54) [reply imdb.py]: @akshatnitd

**Pull request closed without merging**